{"name": "test_for_users_update", "status": "failed", "statusDetails": {"message": "AssertionError: \n            Expected response status codes: (200,)\n            Actual status code: 500", "trace": "@allure.feature(\"Update a User\")\n    @allure.story(\"Authorized user can update users information\")\n    @allure.severity(allure.severity_level.BLOCKER)\n    def test_for_users_update():\n    \n        response = helper.put_request(url=PUT_URL + '/' + str(user_id),\n                                              body={'name':'Abhishek Yadav', 'gender':'Male',\n                                                    'email':'yabhishek@outlook.com', 'status':'Inactive'})\n    \n>       check.response_has_status_code(response, 200)\n\nresponse   = <Response [500]>\n\ntest.py:100: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse = <Response [500]>, status_codes = (200,), actual_status_code = 500\n\n    def response_has_status_code(response, *status_codes):\n        with allure.step(f'Check response has one of {status_codes} status code'):\n            actual_status_code = response.status_code\n            assert actual_status_code in status_codes, f'''\n                Expected response status codes: {status_codes}\n                Actual status code: {actual_status_code}\n>           '''\nE           AssertionError: \nE                       Expected response status codes: (200,)\nE                       Actual status code: 500\n\nactual_status_code = 500\nresponse   = <Response [500]>\nstatus_codes = (200,)\n\n..\\testlib\\check.py:11: AssertionError"}, "steps": [{"name": "Send PUT request by authorized user to url:\n https://gorest.co.in/public-api/users/1394", "status": "passed", "attachments": [{"source": "dce1a589-9225-4437-a8af-f1358bce94fd-attachment.attach"}, {"source": "0f578773-f34a-4413-80c1-0eec62810b72-attachment.attach"}], "start": 1620599902351, "stop": 1620599902752}, {"name": "Check response has one of (200,) status code", "status": "failed", "statusDetails": {"message": "AssertionError: \n            Expected response status codes: (200,)\n            Actual status code: 500\n        \n", "trace": "  File \"D:\\MyWorkspace\\api_tests_python_allure\\testlib\\check.py\", line 11, in response_has_status_code\n    '''\n"}, "start": 1620599902753, "stop": 1620599902753}], "start": 1620599902351, "stop": 1620599902753, "uuid": "b90e3127-6e85-48ef-a47f-7618e562ac0b", "historyId": "6b3054cb62dc7a55307a9051117daa2e", "testCaseId": "551dd67b8ae93505bce81ea5c840d9b7", "fullName": "tests.test#test_for_users_update", "labels": [{"name": "severity", "value": "blocker"}, {"name": "story", "value": "Authorized user can update users information"}, {"name": "feature", "value": "Update a User"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test"}, {"name": "host", "value": "abhishekkyd"}, {"name": "thread", "value": "2756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test"}]}